src/content/zh-tw/_project.yaml

```diff
@@ -0,0 +1,22 @@
+---
+announcement:
+  description: </#cdata-section></style><style><#cdata-section> 錯過了今年的 <b>Chrome 开发峰会</b>?
+    您可以在我們的YouTube播放列表觀看全部的峰会視頻。 <a href="\"https://www.youtube.com/playlist?list=PLNYkxOF6rcICUD5nBfRdAR6Fveosnqa5m\""
+    class="\"gc-analytics-event\"" data-category="\"webFu\"" data-label="\"cds-watch-playlist\"">观看</a>
+buganizer_id: '180451'
+color: google-blue
+content_license: cc3-apache2
+footer_path: "/web/_footer.yaml"
+google_analytics_ids:
+- UA-52746336-1
+home_url: "/web/"
+icon:
+  path: "/web/images/web-fundamentals-icon192x192.png"
+is_family_root: t
```

Comment:
`t` instead of `true`
@sotayamashita

Reply:
@agektmr Sorry for the inconvenience, I’ll keep you posted when we figure out how to handle the Boolean type. https://github.com/gitlocalize/feedback/issues/27